ospfd/ospf_memory.h

```diff
@@ -52,5 +52,7 @@ DECLARE_MTYPE(OSPF_IF_PARAMS)
 DECLARE_MTYPE(OSPF_MESSAGE)
 DECLARE_MTYPE(OSPF_MPLS_TE)
 DECLARE_MTYPE(OSPF_PCE_PARAMS)
+DECLARE_MTYPE(OSPF_EXTERNAL)
+DECLARE_MTYPE(OSPF_REDISTRIBUTE)
```

Comment:
I have removed from ospf_memory[.ch] and moved to ospf_zebra.c as DECLARE_MTYPE_STATIC